content/docs/command-reference/list.md

```diff
@@ -122,3 +122,25 @@ images/dvc-logo-outlines.png.dvc
 images/owl_sticker.png
 ...
 ```
+
+## Example: Archive project data
+
+An archive is a single file that contains multiple files from a project. It can
+be used to backup project data. We can use `dvc list` to create an archive of
+files and directories (<abbr>outputs</abbr>) tracked by DVC:
+
+```dvc
+$ dvc list . -R --dvc-only | zip -@ data.zip  # if `zip` available
+```
+
+Alternative for windows (if `xargs` available):
```

Comment:
- Capital W
- Missing "is"
But I'm not sure this makes sense. How is it an alternative for Windows when it uses `xargs` which is a GNU tool?